src/app.tsx

```diff
@@ -1,21 +1,18 @@
 import InstagramPost from './components/instagramPost/InstagramPost';
-
-import {
-    SAMPLE_POST_IMAGE,
-    SAMPLE_PROFILE_PICTURE,
-    SAMPLE_POST_DESCRIPTION,
-    SAMPLE_USER_NAME,
-    SAMPLE_DATE,
-} from './constants';
+import { MOCK_DATA, POST_IMAGE_SAMPLES, SAMPLES_PROFILE_PICTURE } from './mockText';
```

Comment:
SAMPLE_PROFILE_PICTURE

Reply:
array of profile pictures,
(PROFILE_PICTURE_SAMPLES ?)